sites/cheerpj/src/content/docs/21-migrating-from-cheerpj2.md

```diff
@@ -2,9 +2,9 @@
 title: Migration from CheerpJ 2
 ---
 
-CheerpJ 3 is a complete reimplementation of CheerpJ 2, and as such it is not fully backwards compatible.
+CheerpJ versions 3 and above are a complete reimplementation of CheerpJ 2, and as such, there is no backwards compatibility with CheerpJ 2. This page shows how to migrate from CheerpJ 2 to CheerpJ 3 and above, which is applicable for CheerpJ 4.
```

Comment:
```suggestion
CheerpJ version 3 (and above) is a complete reimplementation of CheerpJ 2, and as such it is not fully backward compatible. This page shows how to migrate from CheerpJ 2 to CheerpJ 3 and above, which is applicable for CheerpJ 4.
```

Most APIs are backward compatible, we should not overstate how many things are visibly changed.